docs/COMPILER_VERSIONS.md

```diff
@@ -63,6 +63,6 @@ We currently use the following version:
 
 | Type                     | Rust version | Note                              |
 | ------------------------ | ------------ | --------------------------------- |
-| Production Rust compiler | 1.69.0       | Builders version 0016             |
-| Min Rust compiler        | 1.67.0       | Supports builder versions >= 0015 |
-| Tooling Rust compiler    | 1.69.0       |                                   |
+| Production Rust compiler | 1.73.0       | Builders version 0017             |
+| Min Rust compiler        | 1.70.0       | Supports builder versions >= 0017 |
+| Tooling Rust compiler    | 1.70.0       |                                   |
```

Comment:
Are these correct like this? I'm not 100% sure

Reply:
Lastest main (without the cosmwasm upgrade) should keep them on 1.69.0. Then bump them along with the CI settings in the other PR.

Reply:
Yeah, makes sense now.